includes/admin/load.php

```diff
@@ -393,3 +396,39 @@ static function ( $name ) {
 		);
 	}
 }
+
+/**
+ * Callback function that print plugin progress indicator script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_print_plugin_progress_indicator_script() {
+	$js_function = <<<JS
+		function addPluginProgressIndicator( message ) {
+			document.addEventListener( 'DOMContentLoaded', function () {
+				document.addEventListener( 'click', function ( event ) {
+					if (
+						event.target.classList.contains(
+							'perflab-install-active-plugin'
+						)
+					) {
+						const target = event.target;
+						target.classList.add( 'updating-message' );
+						target.textContent = message;
+
+						wp.a11y.speak(message);
```

Comment:
Where are we enqueueing the a11y script?

Reply:
Missed it. Added in https://github.com/WordPress/performance/pull/1190/commits/b5e675105396de0e655eeb29903a2d72ebdcad23